dashboard/src/components/Layout/UISelector/Clarity.scss

```diff
@@ -1,6 +1,5 @@
-@import "node_modules/@clr/core/styles/global.min.css";
-@import "node_modules/@clr/city/css/bundles/default.min.css";
-
-body {
-  font-family: "Clarity City", "Avenir Next", "Helvetica Neue", Arial, sans-serif;
+.clr-row {
+  // By default, a row has negative margin. This causes
+  // that a row for the whole body surpasses its limits
+  margin: 0;
```

Comment:
This negative margin is required by the column class. You should always use `Column` inside a `Row`. Removing this margin can cause issues in the Clarity grid system.